doc/mb_compare.md

```diff
@@ -0,0 +1,34 @@
+# MirrorCache (mc) concept vs MirrorBrain (mb) concept
+
+1. Scanning
+1.1
+mb uses daily full scan of all files on all mirrors, without regard of whether files were ever downloaded or changed recently. This may be waste of resources.
+mc scans only those locations which are needed by users and prioritize the scans according to popularity.
+1.2
+It may take up to 24 hours before mb notices that file has gone from a mirror. Clients will be incorrectly redirected and receive 404 error.
+mc is making sure that file exists on selected mirror and never redirects to locations where file was missing for a while.
```

Comment:
so mirrorcache is doing a check request before redirecting?

Reply:
Yes, the thing is that it is easy to do such check with Mojolicious without blocking server resources.